modules/mobile/spec/requests/mobile/v0/messaging/health/all_recipients_spec.rb

```diff
@@ -40,7 +40,9 @@
         care_systems_stub
       )
       VCR.use_cassette('sm_client/triage_teams/gets_a_collection_of_all_triage_team_recipients') do
-        get '/mobile/v0/messaging/health/allrecipients', headers: sis_headers
+        VCR.use_cassette('sm_client/get_unique_care_systems') do
+          get '/mobile/v0/messaging/health/allrecipients', headers: sis_headers
+        end
```

Comment:
probably we could add validation that `healthCareSystemName` is updated as a result of this

Reply:
I see that you actually include this validation in client spec. Do you think we still need it to flag any potential serializer changes that may impact request response  with additional test?